crates/matrix-sdk-test-macros/src/lib.rs

```diff
@@ -38,13 +38,15 @@ pub fn async_test(_attr: TokenStream, item: TokenStream) -> TokenStream {
     let fn_call: TokenStream = if fun.sig.asyncness.is_some() {
         quote! {
             {
-                assert!(#fn_name().await.is_ok());
+                let res = #fn_name().await;
+                assert!(res.is_ok(), "{:?}", res);
```

Comment:
Or just `.unwrap()`?

Reply:
unwrap unfortunately often doesn't give us good enough information about the actual error. It only tells us that we tried to `unwrap` a result and then (b/c of the macro) links to the wrong code location. This assert on the other hand returns debuggable information of the actual cause of the error.